### DISCUSSION TITLE
Welcome to opinix-trade Discussions!

### DISCUSSION THREAD

**Discussion by Praashh** [Praashh/opinix-trade#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HarshitTiwari-20** [comment]
Hi I am Harshit from Corohot 3 and I am here for learning and contributing 😅.

Sometimes I do ask some illogical questions please pardon me

**Reply by Praashh** [reply]
welcome @HarshitTiwari-20
